.travis.yml

```diff
@@ -8,32 +8,38 @@ language: python
 
 matrix:
     include:
-        - python: 2.7
+        - name: "Run test suite on Linux with Python 2.7"
           os: linux
+          python: 2.7
           env: TOXENV=py27
-        - python: 2.7
+        - name: "Run test suite on Linux with Python 3.6"
           os: linux
+          python: 3.6
+          env: TOXENV=py36
+        - name: "Check code style with pylama on Linux - Python 3.6"
+          os: linux
+          python: 3.6
           env: TOXENV=pylama
-        - language: generic
-          os: osx
-          # 7.2 is OS X 10.11.x
+          # 10.1 is OS X 10.13.x
           # https://docs.travis-ci.com/user/languages/objective-c/#Supported-OS-X-iOS-SDK-versions
-          osx_image: xcode7.2
+        - name: "Run test suite on OS X with Python 2.7"
+          os: osx
+          osx_image: xcode10.1
+          language: generic
+          python: 2.7
           env: TOXENV=py27
+        - name: "Run test suite on OS X with Python 3.6"
+          language: generic
+          os: osx
+          env:
+            - TOXENV=py36
+            - HOMEBREW_NO_AUTO_UPDATE=1
```

Comment:
It is strange, but if I do not specify the `HOMEBREW_NO_AUTO_UPDATE=1`, the Travis build with Py3 fails, because of `brew update`.
Here is a job where I removed the `HOMEBREW_NO_AUTO_UPDATE=1` option: https://travis-ci.org/mozilla/mozdownload/jobs/503484085

Reply:
I assume that `brew update` is automatically run? From the log I can see that Python is already installed: `Error: python 3.6.5_1 is already installed`, which comes from `brew install python` in `.travis/install.sh`. Is Python3.6 not installed by default?

Reply:
I removed the `HOMEBREW_NO_AUTO_UPDATE=1` and the Python install command.
Not it should be more cleaner.